server/storage/storage_test.go

```diff
@@ -434,6 +434,9 @@ func testGetChanges(t *testing.T, s MetaStore) {
 	if _, ok := s.(RethinkDB); ok {
 		require.Error(t, err1)
 		require.Error(t, err2)
+	} else if _, ok := s.(CouchDB); ok {
```

Comment:
It might be clearer to convert this `if/else if/else` block at this point to a `switch` statement based on the type of `s`, at this point.  E.g.:

```golang
switch s.(type) {
case RethinkDB: ...
case CouchDB: ...
default: ...
}
```

Reply:
Fixed.